src/v/pandaproxy/rest/handlers.cc

```diff
@@ -283,7 +290,7 @@ create_consumer(server::request_t rq, server::reply_t rp) {
     };
 
     co_return co_await rq.service().client().invoke_on(
```

Comment:
@BenPope it feels a bit awkward that `rq.service()` occurs after `rq` is moved from into the lambda capture above. I take there isn't anything to worry about?

Reply:
#3628 fixes this